### PR TITLE
fix?(content): HR Streamline events that change systems

### PR DESCRIPTION
**Pull request by MasterOfGrey**
With the new remove object feature it's possible to edit systems *much* more smoothly than before.

Due to a lack of understanding of exactly how it worked only a couple parts were updated previously so as not to break anything more than was already the case; this addresses the rest of the events.

Credit to @warp-core who created the feature and originally PR'd *most* of this to my branch with no testing. I only had to fix issues with 2 events to make it actually work as intended.

### CODE REVIEW

**Comment by warp-core** [comment]
Note that doing this this way now will probably break things for people who got some but not all the events already. ~~Not that that mattered before.~~

**Comment by Hurleveur** [comment]
I say let's not worry about it unless if it was in a release.

**Comment by MasterOfGrey** [comment]
> Note that doing this this way now will probably break things for people who got some but not all the events already.

When the wormhole sprite changes it should clean-up any of the possible weirdness leftover.

File: data/hai/hai reveal 3 early.txt [hunk 1]
```diff
@@ -194,31 +194,10 @@ mission "Hai Building Mountaintop [1b] Return"
...
-			period 557.553
-			object
-				sprite planet/ice8-b
-				distance 216
-				period 11.1964
-		object "Wormhole Alpha"
-			sprite planet/wormhole
+		remove object
+			distance 4208.63
+			period 2158.5
```
Comment (warp-core) [inline data/hai/hai reveal 3 early.txt:199]: Isn't this the first event? ~~Please move this event closer to where it is actually called, I don't know why anyone would suggest moving it here while still using it in the first file but any such suggestions were very wrong. But do that in a different PR so the changes made to this event in this one can still be reviewed easily.~~

Comment (MasterOfGrey) [reply data/hai/hai reveal 3 early.txt:199]: Good catch, I'll fix that tonight.

Comment (MasterOfGrey) [reply data/hai/hai reveal 3 early.txt:199]: Nope, nevermind, the first event actually is in HR1 - there's one mission here in HR3 that fires an event when you accept the mission *and* an event when you complete the mission.

This is why these two are together, they're used in the same mission.
